### ISSUE TITLE
[NEW] Adding to user icons to start a new direct message

### ISSUE THREAD

**Issue by ishiikurisu** [RocketChat/Rocket.Chat.Android#328]
Do you know what could be a good feature? Clicking on a user's icon on the chat to direct message them. I don't know if this is already implemented and I can't use it and tell my friends, but this is something I would love to add to this app. Is it possible?

**Comment by rafaelks** [comment]
@ishiikurisu Yes, that's a good idea, thanks for sharing with us. This is totally possible, but not implemented yet. I think it will be implemented in the future as a context menu, with other options, such like: "message user", "pin message", etc.

**Comment by rafaelks** [comment]
@ishiikurisu I've assigned the issue to you, so you can start playing with it.

**Comment by ishiikurisu** [comment]
@rafaelks ok!
